crates/subspace-gateway/Cargo.toml

```diff
@@ -20,6 +20,7 @@ include = [
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
+actix-web = "4"
```

Comment:
I think we might need to make the actix and reqwest dependencies consistent using features. We shouldn’t be using both `native-tls` and `openssl-sys` (unless it’s for different platforms).

Also, from a security perspective, it would be better to avoid OpenSSL.

Reply:
I’d suggest using the `rustls-tls` / `rustls` features, and setting default features to false. That will minimise unnecessary dependencies.

@clostao do you need any of these features or compression methods?
- reqwest: "charset" (Unicode), "http2"
- actix-web: "compress-brotli", "compress-gzip", "compress-zstd", "cookies", "http2", "unicode"

Reply:
I think that for the current implementation the compression is not needed since IPLD nodes would be <64kb. If we implement batched maybe a good option, but for the moment is not needed imo.

About `unicode` I'd guess that we don't need that either since object mapping content are format-less binary arrays. Not too sure about this one though I might be missing sth.

The `cookies` is not needed at this moment.